plugins/video-toggle/button-switcher.css

```diff
@@ -18,6 +22,7 @@
     color: #fff;
     padding-right: 120px;
     position: absolute;
+    left: calc(50% - 120px);
```

Comment:
It works but (opinionated) I preferred having it on the side, it covers less the video 😄 Maybe we can have it as an additional css file, and make it a plugin option to center or not the toggle? This way users have a choice